src/app/vendors/VendorItems.m.scss

```diff
@@ -1,60 +1,53 @@
 @use '../variables.scss' as *;
 
 .currencies {
```

Comment:
On desktop, you could trim some inconsistent/redundant vertical spacing by absolutely positioning the currency in the top right of the collapsible section to it sits in the whitespace adjacent to sub-headings. 

Atm the vertical spacing is inconsistent if a vendor has few offerings or you're on a wide resolution, sometimes there's extra spacing, sometimes not. 

Current:
![image](https://github.com/DestinyItemManager/DIM/assets/1396158/8a2a5908-6282-427b-acc8-45ab3e44a3e6)

Absolute position currency:
![image](https://github.com/DestinyItemManager/DIM/assets/1396158/6e276446-0e1d-4255-82a3-db665586729e)



Mobile would need to stay as-is for Rahool who has a tonne of currencies that overlap 
![image](https://github.com/DestinyItemManager/DIM/assets/1396158/a2448f0a-6be0-4430-ba8e-7f700191d23e)

Reply:
Yeah, I've tried that before, but it doesn't really work:

<img width="520" alt="Screenshot 2023-11-11 at 2 05 54 PM" src="https://github.com/DestinyItemManager/DIM/assets/313208/97a71123-00a7-4024-bc3e-2cd70ea180a8">